lib/modules/selectedEntry.js

```diff
@@ -375,50 +345,49 @@ export function move(direction: $Keys<typeof movers>, options?: SelectOptionsWit
 	refreshKeyMoveTimer();
 }
 
-// old style: .RES-keyNav-activeElement { '+borderType+': '+focusBorder+'; background-color: '+focusBGColor+'; } \
-// this new pure CSS arrow will not work because to position it we must have .RES-keyNav-activeElement position relative, but that screws up image viewer's absolute positioning to
-// overlay over the sidebar... yikes.
-// .RES-keyNav-activeElement:after { content: ""; float: right; margin-right: -5px; border-color: transparent '+focusBorderColor+' transparent transparent; border-style: solid; border-width: 3px 4px 3px 0; } \
-
 // why !important on .RES-keyNav-activeElement?  Because some subreddits are unfortunately using !important for no good reason on .entry divs...
 
-function addFocusBGColor() {
-	const focusBGColor = module.options.focusBGColor.value || module.options.focusBGColor.default;
-	const focusBGColorNight = module.options.focusBGColorNight.value || module.options.focusBGColorNight.default;
-	const focusFGColorNight = module.options.focusFGColorNight.value || module.options.focusFGColorNight.default;
-
+function styleLine() {
 	addCSS(`
+		.RES-keyNav-activeElement { box-shadow: 3px 0 0 -1px #c2d2e0; !important; }
```

Comment:
Using `box-shadow` instead of `border` or `outline` avoids compressing the element.